library/Reader.php

```diff
@@ -210,7 +210,7 @@ function getHeader($header) {
                     $this->headers['runs']++;
                     $this->headers['summary'] += (double) $subParts[0];
                 } else {
-                    $this->headers[$parts[0]] = $parts[1];
+                    $this->headers[$parts[0]] = $parts[1] ?? 'unknown';
```

Comment:
When I run it with E_ALL error reporting then I had to use
`$this->headers[$parts[0]] = array_key_exists($parts[1]) ? $parts[1] : 'unknown';`

Otherwise I still got the error that the array key doesn't exist.

Reply:
?? operator is a new syntax added in PHP7, array_key_exists($parts[1]) syntax wrong? should be array_key_exists($key, $array)

Reply:
Compatibility of the project is 5, so better to use array_key_exist or empty or whatever you prefer.